ui/components/atoms/A011-Checkbox/schema.js

```diff
@@ -0,0 +1,32 @@
+export default {
+  $schema: 'http://json-schema.org/schema#',
+  type: 'object',
+  properties: {
+    checked: {
+      type: 'boolean'
+    },
+    color: {
+      type: 'string',
+      enum: ['default', 'primary', 'secondary']
+    },
+    disabled: {
+      type: 'boolean'
+    },
+    disabledRipple: {
+      type: 'boolean'
+    },
+    id: {
+      type: 'string'
+    },
+    indeterminate: {
+      type: 'boolean'
+    },
+    value: {
+      type: 'string'
+    },
+    type: {
+      type: 'string'
+    }
+  },
+  required: ['value']
```

Comment:
Why is value required?

Reply:
I'm thinking that we need to put a value of each checkbox..